shared/js/media/crop_resize_rotate.js

```diff
@@ -1,7 +1,9 @@
 /* exported cropResizeRotate */
-/* global getImageSize */
+/* global getImageSize, LazyLoader, JPEG */
 /* global Downsample */
 
+'use strict';
```

Comment:
Since there is only a single function in this file, I prefer to put the use strict inside the function. That way it is guaranteed to be valid.  Otherwise, if this file gets concatenated to another file, the use strict will be ignored.